livespark-showcase/livespark-webapp/pom.xml

```diff
@@ -43,6 +43,12 @@
   </properties>
 
   <dependencies>
+    <!-- drools-compiler has dependency on org.eclipse.jdt.core.compiler:ecj:jar:3.5.1:compile, see http://code.google.com/p/google-web-toolkit/issues/detail?id=4479 -->
```

Comment:
I was thinking much the same thing over at https://github.com/droolsjbpm/drools-wb/pull/263/files#diff-7a8bb56e50632e02768371ed4ddc1688R42